touchforms/formplayer/static/formplayer/script/entrycontrols_full.js

```diff
@@ -1,785 +1,584 @@
-        if (this.map) {
-          this.map.setCenter(new google.maps.LatLng(answer[0], answer[1]));
-          this.map.setZoom(this.DEFAULT.anszoom);
+        if (lat == this.lat && lon == this.lon) {
+            return false;
         }
-      } else {
-        this.set_latlon(null, null);
-      }
-    } else {
-      this.default_answer = answer;
-    }
-  }
 
-  this.update_center = function() {
-    var center = this.map.getCenter();
-    var lat = center.lat();
-    var lon = center.lng();
+        this.lat = lat;
+        this.lon = lon;
 
-    if (this.set_latlon(lat, lon)) {
-      var widget = this;
-      this.delay_action('move', function() { widget.commit(); }, 1.);
-    }
-  }
+        this.$lat.text(lat != null ? (lat >= 0. ? 'N' : 'S') + intpad(Math.abs(lat).toFixed(5), 8) : '??.?????');
+        this.$lon.text(lat != null ? (lon >= 0. ? 'E' : 'W') + intpad(Math.abs(lon).toFixed(5), 9) : '???.?????');
 
-  this.set_latlon = function(lat, lon) {
-    lon = lon % 360.;
-    if (lon < 0) {
-      lon += 360.
-    }
-    lon = lon % 360.;
-    if (lon >= 180.) {
-      lon -= 360.;
+        return true;
     }
 
-    if (lat == this.lat && lon == this.lon) {
-      return false;
+    this.delay_action = function(tag, dofunc, delay) {
+        var timer = this.timers[tag];
+        if (timer) {
+            clearTimeout(timer);
+            timer = null;
+        }
+        this.timers[tag] = setTimeout(dofunc, 1000 * delay);
     }
 
-    this.lat = lat;
-    this.lon = lon;
-
-    this.$lat.text(lat != null ? (lat >= 0. ? 'N' : 'S') + intpad(Math.abs(lat).toFixed(5), 8) : '??.?????');
-    this.$lon.text(lat != null ? (lon >= 0. ? 'E' : 'W') + intpad(Math.abs(lon).toFixed(5), 9) : '???.?????');
-
-    return true;
-  }
-
-  this.delay_action = function(tag, dofunc, delay) {
-    var timer = this.timers[tag];
-    if (timer) {
-      clearTimeout(timer);
-      timer = null;
+    this.search = function(query) {
+        var map = this.map;
+        this.geocoder.geocode({
+            'address': query
+        }, function(results, status) {
+            if (status == google.maps.GeocoderStatus.OK) {
+                map.fitBounds(results[0].geometry.viewport);
+                map.setCenter(results[0].geometry.location);
+            }
+        });
     }
-    this.timers[tag] = setTimeout(dofunc, 1000 * delay);
-  }
-
-  this.search = function(query) {
-    var map = this.map;
-    this.geocoder.geocode({'address': query}, function(results, status) {
-      if (status == google.maps.GeocoderStatus.OK) {
-        map.fitBounds(results[0].geometry.viewport);
-        map.setCenter(results[0].geometry.location);
-      }
-    });
-  }
 }
+GeoPointEntry.prototype = Object.create(Entry.prototype);
+GeoPointEntry.prototype.constructor = Entry;
 
-function renderQuestion (q, $container, init_answer) {
-  var control = null;
-
-  q.domain_meta = q.domain_meta || {};
-
-  if (q.customlayout != null) {
```

Comment:
i'm pretty sure this was only used by the touchscreen version to do things like render a special widget for blood pressure